benchmark/king2dcompressible/ala/ala_Di025_Ra4.prm

```diff
@@ -0,0 +1,23 @@
+set End time                               = 1.5
+set Output directory                       = Di025_Ra4
+
+subsection Adiabatic conditions model
+  subsection Function
+    set Function constants  = Di=0.25, gamma=1.0
+    set Function expression = 0.091*exp(depth*Di); gamma/Di*(exp(depth*Di/gamma)-1); exp(depth*Di/gamma)  # T,p,rho
+    set Variable names      = depth
+  end
+end
+
+subsection Material model
+  subsection Tan Gurnis model
```

Comment:
Does it really use the Tan-Gurnis model, or was that just a convenient starting point?